src/plugins/Wallet/api.ts

```diff
@@ -118,11 +118,11 @@ export const redPacketAPI = {
                     name,
                     token_type,
                     token_addr,
-                    total_tokens.toString(),
+                    total_tokens.toFixed(),
```

Comment:
plz set `digits`.

see https://mdn.io/Number.prototype.toFixed

Reply:
It is treated as 0 by default.